pages/linux/quotacheck.md

```diff
@@ -0,0 +1,28 @@
+# quotacheck
+
+> Scan a filesystem for disk usage; create, check and repair quota files.
+> It is best to run quota check with quotas turned off to prevent damage or loss to quota files.
+
+- Check all mounted non-NFS filesystems:
+
+`sudo quotacheck --all`
+
+- Force check even if quotas are enabled:
+
+`sudo quotacheck --force {{filesystem}}`
```

Comment:
Its a bit unclear what `filesystem` refers to here. Does it mean a filesystem type (ext3,ext4,xfs) ? Or a filesystem partition (/dev/sda5,/dev/sda6) ? 

It most probably means the latter. It would be good if we can mention that with an example rather than just saying `filesystem`. Makes everything instantly clear.